packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -580,6 +580,10 @@ type MessagesType = {
   'lockbox.service.messages.storagespace': 'Insufficient storage space on device'
   'lockbox.service.messages.unknown': 'An unknown error has occurred.'
   'lockboxrequired.tooltip': 'Hardware device is required'
+  'modals.brokerage.bank_preview.disclaimer': 'This account can be used for buys, sells & withdrawals.'
+  'modals.brokerage.remove_bank.description': "You'e about to remove your {bankAccount}"
+  'modals.brokerage.remove_bank.back': 'Back'
+  'modals.brokerage.curency': 'Curency'
```

Comment:
typo here in `Currency`. also create maybe a create a generic `copy.currency` item if it doesnt already exist